elasticassignment/ageofempires/Elastic_search/elastic_client.py

```diff
@@ -0,0 +1,12 @@
+import requests
+import elasticsearch
+from django.conf import settings
+
+class elastic_client:
+
+    def make_connection():
+        return elasticsearch.Elasticsearch([settings.CONFIG])
+
+    def get_data(url:str):
+        response_data = requests.get(url)
+        return response_data.json()
```

Comment:
The functionality in this function is not related to Elastic Client. 

For HTTP calls please create a separate client class `HTTPClient` with `get` method.